changelogs/current.yaml

```diff
@@ -1,9 +1,22 @@
 date: Pending
 
 behavior_changes:
+<<<<<<< HEAD
```

Comment:
this and === and >>> lines below seem to be a merge conflict leftover. Harmless in this pr as it's just doc. Can be fixed later.